src/index.html

```diff
@@ -6,7 +6,316 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo-link">
+          <img
+            class="header__logo-img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="wrapper-card">
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          >
+          <div class="card__info">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rate">
+              <div class="stars">
```

Comment:
you need to add stars modifier here, please check the checklist
![Screenshot 2023-06-03 at 19 00 06](https://github.com/mate-academy/layout_catalog/assets/64746943/de912bed-cb69-4c42-9386-38eda78d1594)